src/language-js/utils/index.js

```diff
@@ -405,6 +405,16 @@ function isTestCall(node, parent) {
 /** @return {(node: Node) => boolean} */
 const skipChainExpression = (fn) => (node) => {
   if (node?.type === "ChainExpression") {
+    // @ts-expect-error
+    if (node.object == null) {
+      // @ts-expect-error
+      node.object = node.expression.object;
+    }
+    // @ts-expect-error
+    if (node.property == null) {
+      // @ts-expect-error
+      node.property = node.expression.property;
+    }
```

Comment:
@fisker This isn't a good. I'm merging this PR as a temporary solution, but do you have any better solution?

Reply:
Yes, this is wrong, should update logic where it's accessing the object/property.

Reply:
This may also cause `computed`/`shorthand` check part result wrong..